membrane_curvature/tests/test_mdakit_membcurv.py

```diff
@@ -163,24 +163,121 @@
+               13932, 13944, 13956, 13968, 13980, 13992, 14004, 14016, 14028, 14040,
+               14052, 14064, 14076, 14088, 14100, 14112, 14124, 14136, 14148, 14160,
+               14172, 14184, 14196, 14208, 14220, 14232, 14244, 14256, 14268, 14280],
+
+      'POPE': [14292, 14304, 14316, 14328, 14340, 14352, 14364, 14376, 14388, 14400, 14412,
+               14424, 14436, 14448, 14460, 14472, 14484, 14496, 14508, 14520, 14532, 14544,
+               14556, 14568, 14580, 14592, 14604, 14616, 14628, 14640, 14652, 14664, 14676,
+               14688, 14700, 14712, 14724, 14736, 14748, 14760, 14772, 14784, 14796, 14808,
+               14820, 14832, 14844, 14856, 14868, 14880, 14892, 14904, 14916, 14928, 14940,
+               14952, 14964, 14976, 14988, 15000, 15012, 15024, 15036, 15048, 15060, 15072,
+               15084, 15096, 15108, 15120, 15132, 15144, 15156, 15168, 15180, 15192, 15204,
+               15216, 15228, 15240, 15252, 15264, 15276, 15288, 15300, 15312, 15324, 15336,
+               15348, 15360, 15372, 15384, 15396, 15408, 15420, 15432, 15444, 15456, 15468,
+               15480, 15492, 15504, 15516, 15528, 15540, 15552, 15564, 15576, 15588, 15600,
+               15612, 15624, 15636, 15648, 15660, 15672, 15684, 15696, 15708, 15720, 15732,
+               15744, 15756, 15768, 15780, 15792, 15804, 15816, 15828, 15840, 15852, 15864,
+               15876, 15888, 15900, 15912, 15924, 15936, 15948, 15960, 15972, 15984, 15996,
+               16008, 16020, 16032, 16044, 16056, 16068, 16080, 16092, 16104, 16116, 16128,
+               16140, 16152, 16164, 16176, 16188, 16200, 16212, 16224, 16236, 16248, 16260,
+               16272, 16284, 16296, 16308, 16320, 16332, 16344, 16356, 16368, 16380, 16392,
+               16404, 16416, 16428, 16440, 16452, 16464, 16476, 16488, 16500, 16512, 16524,
+               16536, 16548, 16560, 16572, 16584, 16596, 16608, 16620, 16632, 16644, 16656,
+               16668, 16680, 16692, 16704, 16716, 16728, 16740, 16752, 16764, 16776, 16788,
+               16800, 16812, 16824, 16836, 16848, 16860, 16872, 16884, 16896, 16908, 16920,
+               16932, 16944, 16956, 16968, 16980, 16992, 17004, 17016, 17028, 17040]}},
+
+    'small': {'lower': {'POPC': [5, 6, 7], 'POPE': [8, 9]}, 'upper': {'POPC': [0, 1, 4], 'POPE': [2, 3]}},
+
+    'med': {'lower': {'POPC': [12, 13, 14, 15, 16],
+                      'POPE': [17, 18, 19, 20, 21, 22, 23, 24]},
+            'upper': {'POPC': [0, 1, 2, 3, 4, 5, 6, 7, 11],
+                      'POPE': [8, 9, 10]}},
+
+    'big': {'lower': {'POPC': [25, 26, 27, 28, 29, 30, 31, 32, 33, 34],
+                      'POPE': [35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49]},
+            'upper': {'POPC': [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 23, 24],
+                      'POPE': [15, 16, 17, 18, 19, 20, 21, 22]}}
 }
 
 
-@pytest.fixture()
+@ pytest.fixture()
```

Comment:
I don't think it's normally conventional to put a space after the @?

Reply:
Interesting. I just noticed it's the pep8 plugin (in VS) adds those spaces. Deleted in the last commit.

Reply:
It might think (wrongly) that `@` is the binary matrix multiplication operator??

In MDAnalysis we typically look how we wrote existing code and then try to match the existing code, unless it appears to go against our [style guide](https://userguide.mdanalysis.org/dev/contributing_code.html?highlight=style%20guide#code-formatting-in-python)